google/cloud/security/common/gcp_api/_base_repository.py

```diff
@@ -41,6 +41,8 @@
 
 LOGGER = log_util.get_logger(__name__)
 
+DEFAULT_HTTP_TIMEOUT = 30.0
+
```

Comment:
super-nit (optional): The globals aren't consistently alpha-sorted and consistently named, e.g. `NUM_HTTP_RETRIES` is a default too but isn't prefixed with `DEFAULT_`. It'd be nice to have this cleaned up.